src/search/landmarks/landmark_factory_h_m.cc

```diff
@@ -350,70 +352,70 @@ void LandmarkFactoryHM::print_pm_op(const VariablesProxy &variables, const PMOp
         cond_eff.clear();
         int pm_fluent;
         size_t j;
```

Comment:
Do I set it correctly that this function is only called within a "log.at_least_X" condition? If yes that is good, otherwise, I am wondering if we always want to log this.

Reply:
This is basically the same thing again: do we want to if-guard all `print_*` methods or do we say that whoever calls such a method should know that this method will produce output. I think this is a question we could discuss with the others.